python/GafferImageUI/ImageViewUI.py

```diff
@@ -491,6 +589,32 @@ def _plugDirtied( self, childPlug ):
 			# TODO - should GafferUI.Image have a setImage?
 			self.__modeImage._qtWidget().setPixmap( GafferUI.Image._qtPixmapFromFile( [ "sourceCursor.png", "sourcePixel.png", "sourceArea.png" ][ mode ]  ) )
 			self.__updateLazily()
+		elif childPlug == self.getPlug().node()["colorInspector"]["inspectorCompareMode"]:
+			inspectorCompareMode = self.getPlug().node()["colorInspector"]["inspectorCompareMode"].getValue()
+			if self.__compareMode:
+				self.__compareMode.setImage( self.__compareModeIconDict[ inspectorCompareMode ] )
+			self.__updateCompareMode()
+		elif childPlug == self.getPlug().node()["compare"]["mode"]:
+			compareMode = self.getPlug().node()["compare"]["mode"].getValue()
+			if self.__compareMode:
+				self.__compareMode.setVisible( compareMode != "" )
+			if self.__compareModeSpacer:
+				self.__compareModeSpacer.setVisible( compareMode != "" )
```

Comment:
Perhaps `setEnabled()` rather than `setVisible()`, so there's a bit less jumping around and relayouting?

Reply:
Feature removed.